async-query/build.gradle

```diff
@@ -99,7 +99,8 @@ jacocoTestCoverageVerification {
                     // ignore because XContext IOException
                     'org.opensearch.sql.spark.execution.statestore.StateStore',
                     'org.opensearch.sql.spark.rest.*',
-                    'org.opensearch.sql.spark.scheduler.OpenSearchRefreshIndexJobRequestParser',
+                    'org.opensearch.sql.spark.scheduler.parser.OpenSearchScheduleQueryJobRequestParser',
+                    'org.opensearch.sql.spark.scheduler.parser.IntervalScheduleParser',
```

Comment:
We don't want to exclude unless there is special reason.

Reply:
the opensearch runner was included becasuse of xcontext ioexception, removed IntervalScheduleParser